hypothesis-python/src/hypothesis/stateful.py

```diff
@@ -17,15 +17,19 @@
 """
 import collections
 import inspect
+import trio
```

Comment:
This needs to be a conditional import, so we only actually import `trio` when the user explicitly opts in to some Trio-specific functionality.

I suggest starting with the `asyncio` version for now.